Sources/Controllers/Panels/OAMapOpacitySliderToggler.h

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
I think such class is not needed. Would be better to call updateOverlayUnderlayView at OAOverlayMapLayer / OAUnderlayMapLayer in updateLayer method and in onShowSwitchChanged method. The updateOverlayUnderlayView should be without parameter since we introduce new settings.

Reply:
I got it. I'll remake it.

Reply:
Done.